src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -102,10 +102,13 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 			if len(seeds) == 0 {
 				// If --seeds wasn't passed, fall back to the
 				// env var.
-				seeds = strings.Split(
+				envSeeds := strings.Split(
 					os.Getenv("REDPANDA_SEEDS"),
 					",",
 				)
+				if len(envSeeds) != 0 {
+					seeds = envSeeds
+				}
 			}
 			seedServers, err := parseSeeds(seeds)
```

Comment:
What will happen if `seeds` aren't passed from neither `--seeds` nor the environment ?

Reply:
Nothing. `parseSeeds` will return an empty `[]config.SeedServer`: https://github.com/vectorizedio/redpanda/blob/e9f65fad3299f478ce475a96bd326abc2d192c0b/src/go/rpk/pkg/cli/cmd/start.go#L633